supervisor/resolution/checks/plugin_trust.py

```diff
@@ -35,6 +35,9 @@ async def run_check(self) -> None:
                 self.sys_resolution.create_issue(
                     IssueType.TRUST, ContextType.PLUGIN, reference=plugin.slug
                 )
+            except CodeNotaryBackendError:
+                _LOGGER.warning("CAS backend issue, skipping check")
+                return
```

Comment:
```suggestion
```
There is nothing more after this, so no need to return here

Reply:
it's on a loop

Reply:
Ah, right 👍